src/midi.js

```diff
@@ -36,7 +47,17 @@ const midi = (notes, fileName) => {
 		}
 	});
 
-	fs.writeFileSync(fileName, file.toBytes(), 'binary');
+	let outputFile = path.join(process.cwd(), '/Output/', fileName + '.mid');
+
+	writeFile(outputFile, file.toBytes(), 'binary');
 }
 
-module.exports = midi;
+function writeFile(outPath, contents, options) {
+	  let outputPathCreated = mkdirp.sync(getDirName(outPath));	   
+	  if (outputPathCreated || outputPathCreated === null) {
```

Comment:
Can you help me understand this `if` condition? It seems to mean, even if `outputPathCreated` has a value or not, continue writing the file.